_shows/19_20/still_alice.md

```diff
@@ -79,16 +100,18 @@ crew:
 - role: Technical Operator
   name: Rose Edgeworth 
 - role: Technical Operator
-  name: Alice Clothier 
-
+  name: Alice Clothier
+- role: Technical Operator
+  name: Danni Cooper 
 
 prod_shots: x24nSs
 
 assets:
   - type: poster
     image: fmpWm9s
-
-# Programme - https://drive.google.com/file/u/3/d/1PgJRxNl79t-asv27p20gZBAvpYfhRJJD/preview
+  - type: programme
+    title: Still Alice Programme
+    filename: still-alice-programme.pdf
```

Comment:
This 404s

Reply:
it's in the LFS and references the correct filename... :(